ext/nnstreamer/tensor_filter/tensor_filter_deepview_rt.cc

```diff
@@ -0,0 +1,594 @@
+/* SPDX-License-Identifier: LGPL-2.1-only */
+/**
+ * NNStreamer tensor_filter, sub-plugin for DeepView RT
+ * Copyright 2022 NXP
+ */
+/**
+ * @file    tensor_filter_deepview_rt.cc
+ * @date    7 Jun 2022
+ * @brief   NNStreamer tensor-filter sub-plugin for DeepView RT engine
+ * @see     http://github.com/nnstreamer/nnstreamer
+ * @author  Julien Vuillaumier <julien.vuillaumier@nxp.com>
+ * @bug     No known bugs except for NYI items
+ */
+#include <fstream>
+#include <limits.h>
+#include <memory>
+#include <stdexcept>
+#include <thread>
+#include <unistd.h>
+#include <vector>
+
+
+#define DVRT_SUBPLUGIN_NAME "deepview-rt"
+
+
+/**
+ * @brief Glib logging domain
+ *        set environment variable G_MESSAGES_DEBUG=deepview-rt for debug/info logs
+ */
+#define G_LOG_DOMAIN DVRT_SUBPLUGIN_NAME
+
+#include <glib.h>
+#include <nnstreamer_cppplugin_api_filter.hh>
+#include <nnstreamer_log.h>
+#include <nnstreamer_util.h>
+#include <tensor_common.h>
+
+#include <deepview_rt.h>
+
+using namespace std;
+
+
+namespace nnstreamer
+{
+namespace tensorfilter_dvrt
+{
+#ifdef __cplusplus
+extern "C" {
+#endif /* __cplusplus */
```

Comment:
This is C++ code. Checking __cpluccplus is meaningless.

Besides, ```_init_filter_dvrt()``` is defined in this .cc file; thus, ```_init_filter_dvrt()``` is compiled by C++ compiler, not C compiler. Therefore, they are C++ function; ```extern "C"``` is not needed and should not be applied.

Reply:
Agreed, ifdef __cplusplus statement is not necessary
Test removed in updated patchset.